canonical-websites/www.ubuntu.com
# Kubernetes managed page build

## Done
Update the content on the k8s managed page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes/managed](http://0.0.0.0:8001/kubernetes/managed)
- Check that the page matches the [copy doc](https://docs.google.com/document/d/15jzhasokjH5Ql9Za56ZxRknXNNhEXYdIFXXtLODGxns/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3357
